Realm/RLMSyncManager.mm

```diff
@@ -150,47 +151,31 @@ - (void)_fireErrorWithCode:(int)errorCode
                   userInfo:(NSDictionary *)userInfo
                 errorClass:(RLMSyncSystemErrorKind)errorClass {
     NSError *error = nil;
-    NSMutableDictionary *mutableUserInfo = [userInfo mutableCopy];
-    mutableUserInfo[@"description"] = message;
-    mutableUserInfo[@"error"] = @(errorCode);
-    mutableUserInfo[@"underlying_class"] = @(errorClass);
-
+    BOOL shouldMakeError = YES;
+    NSDictionary *custom = nil;
     switch (errorClass) {
         case RLMSyncSystemErrorKindClientReset: {
-            // Client reset is a special case; the application can respond to it to a greater degree than
-            // it can for most other errors.
-            mutableUserInfo[kRLMSyncPathOfRealmBackupCopyKey] = userInfo[@(realm::SyncError::c_recovery_file_path_key)];
+            // Client reset is a special case; the application
```

Comment:
Wording is a bit awkward here. Consider rewording:

```
// Users can respond to "client reset" errors to a
// greater degree than it can for most other errors.
```